src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -1614,6 +1618,21 @@ public CaseDataBuilder generatePaymentDateForAdmitPartResponse() {
         return this;
     }
 
+    public CaseDataBuilder generateRepaymentDateForAdmitPartResponse() {
+        atStateRespondentRespondToClaimFastTrack(RespondentResponseType.PART_ADMISSION);
+        respondent1ClaimResponseDocument = ResponseDocument.builder()
+            .file(DocumentBuilder.builder().documentName("defendant-response.pdf").build())
+            .build();
+        respondent1DQ();
+
+        respondent1ClaimResponseTypeForSpec = RespondentResponseTypeSpec.PART_ADMISSION;
```

Comment:
Why is this used?

Reply:
PART_ADMISSION is used because this page is part of Admit claim (PART_ADMISSION) journey 
"PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND defenceAdmitPartPaymentTimeRouteRequired=\"SUGGESTION_OF_REPAYMENT_PLAN\"",

Reply:
resolved